### PR TITLE
commands to see the branch deployed on the server

### PR DESCRIPTION
**Pull request by RahulMahale**

### CODE REVIEW

**Comment by RahulMahale** [comment]
@neerajdotname once you merge this, also merge https://github.com/bigbinary/boxy-cap/pull/13

File: docs/how-to.md [hunk 1]
```diff
@@ -89,3 +89,17 @@ If tail does not support tailing multiple files then install MultiTail. https://
 ```
 ps aux --sort=-%mem | awk 'NR<=10{print $0}'
 ```
+
+
+### How do I find which branch is deployed on server
```
Comment (neerajdotname) [inline docs/how-to.md]: Change it to

```
How do I find what all commits are deployed on server
``

File: docs/how-to.md [hunk 2]
```diff
@@ -89,3 +89,17 @@ If tail does not support tailing multiple files then install MultiTail. https://
...
+
+check last 5 deployed branches on the server using
```
Comment (neerajdotname) [inline docs/how-to.md]: `Check last 5 commits deployed on the server`

**Comment by neerajdotname** [comment]
@RahulMahale every where please change branches to commits. I want to know the most recent commit to a branch and not branches.

**Comment by RahulMahale** [comment]
@neerajdotname done

**Pull request merged**